drivers/esp-hosted/esp_hosted_bthci.c

```diff
@@ -126,7 +126,7 @@ int mp_bluetooth_hci_controller_init(void) {
         if (mp_bluetooth_hci_uart_any()) {
```

Comment:
Bit off topic, but @iabdalkader does this just block for 2.5 seconds at boot every time? Does the esp chip take that long to boot, or is it just extra timeout in case?

Reply:
Yes, because the firmware is configured to output debug messages on boot.

Reply:
I think you meant to comment on this line: https://github.com/micropython/micropython/blob/acecb0f81909dc02f4ce931ac29a2fe6c58a6818/drivers/esp-hosted/esp_hosted_bthci.c#L125

Reply:
Thanks, I did, but github only seems to let me start a comment on a line x rows from an actual change... this was as close as I could get :-D